lib/index.js

```diff
@@ -75,6 +80,17 @@ info: Storage engine shut down.\n";
       process.stderr.write(output);
     }
   });
+
+  self.Server = server;
+  return server;
+}
+
+Server.prototype.stop = function(cb) {
+  var self = this;
+  self.Server.close(cb);
```

Comment:
Is that safe to do? Isn't closing the server going to make all connections to emit an end/close/error event before you actually has a change to close them?